### DISCUSSION TITLE
疯狂的打灰

### DISCUSSION THREAD

**Discussion by luckey-lang** [share121/inter-knot#203] [其他]
![Screenshot_2024-07-30-13-58-38-505_com miHoYo Nap](https://github.com/user-attachments/assets/0106db0a-f662-4c10-a801-37e9af224ef4)

**Comment by gjbctuj** [comment]
丰矿的王佐

**Comment by yyxzz808** [comment]
啊！打灰！啊！打灰！啊！

**Reply by JinYi-qq** [reply]
本地人头像啊
